poptheme/themes/Pop-Light-color-theme.json

```diff
@@ -1,207 +1,207 @@
 {
-    "name": "Pop Light",
-    "type": "light",
-    "colors": {
-        // Base
-        // "foreground": "",
+	"name": "Pop",
+	"type": "light",
+	"colors": {
+
+		// Base
+		// "foreground": "",
 		"focusBorder": "#574F4A",
-        // "contrastActiveBorder": "",
-        // "contrastBorder": "",
+		// "contrastActiveBorder": "",
+		// "contrastBorder": "",
 
-        // "widget.shadow": "",
-         "textLink.foreground": "#C26C00",
-         "textLink.activeForeground": "#DE8800",
-         "input.background": "#FFFFFF",
-         "input.border": "#0008",
-         "input.foreground": "#403C3A",
-         "input.placeholderForeground": "#403C3A88",
-         "inputOption.activeBorder": "#FAA41A",
-        // "inputValidation.infoBorder": "",
-        // "inputValidation.infoBackground": "",
-        // "inputValidation.warningBackground": "",
-        // "inputValidation.warningBorder": "",
-        // "inputValidation.errorBackground": "",
-        // "inputValidation.errorBorder": "",
+		// "widget.shadow": "",
+		"textLink.foreground": "#C26C00",
+		"textLink.activeForeground": "#DE8800",
+		"input.background": "#FFFFFF",
+		"input.border": "#0008",
+		"input.foreground": "#403C3A",
+		"input.placeholderForeground": "#403C3A88",
+		"inputOption.activeBorder": "#FAA41A",
+		// "inputValidation.infoBorder": "",
+		// "inputValidation.infoBackground": "",
+		// "inputValidation.warningBackground": "",
+		// "inputValidation.warningBorder": "",
+		// "inputValidation.errorBackground": "",
+		// "inputValidation.errorBorder": "",
 
-         "badge.background": "#F15D22",
-         "progressBar.background": "#48B9C7",
+		"badge.background": "#F15D22",
+		"badge.foreground": "#FFF",
+		"progressBar.background": "#48B9C7",
 
-         "dropdown.background": "#F6F6F6",
-         "dropdown.foreground": "#403C3A",
-         "dropdown.border": "#63524b00",
+		"dropdown.background": "#F6F6F6",
```

Comment:
Looks like you've got some tab vs. space issues here. I'd suggest cleaning these up to make a smaller diff that contains *just* the changes you're proposing in this PR.

Reply:
Ah, didn't notice that, thanks for pointing it out!

Reply:
Agreed, once PR is only proposed changes, i'm happy to review and merge.